types/ace/index.d.ts

```diff
@@ -5,10 +5,15 @@
 
 declare namespace AceAjax {
 
+    export interface Point {
+        row: number;
+        column: number;
+    }
+
     export interface Delta {
```

Comment:
I referred to 
https://github.com/ajaxorg/ace/blob/e9f53ddf3c789f0042d83a0c9c3163021d028159/ace.d.ts#L252-L257
and I confirmed this type is correct.
https://github.com/ajaxorg/ace/blob/e9f53ddf3c789f0042d83a0c9c3163021d028159/lib/ace/apply_delta.js#L45-L55